### ISSUE TITLE
How many epochs and what is the best loss of your model in the Chart-Text Alignment stage?

### ISSUE THREAD

**Issue by tannq-jo** [FuxiaoLiu/MMC#11]
Hi,
I am trying to replicate the Chart-Text Alignment stage as described in your paper. It's my first time pre-training such a big model on a large-scale dataset so I don't know when I should stop the pre-training loop. Could you please kindly provide the information about the number of epochs and the best loss that you pre-trained your model in the Chart-Text Alignment stage.  

Thank you for your attention to this matter.

**Comment by FuxiaoLiu** [comment]
I usually train one epoch in both the pertaining and finetuning.
